### PR TITLE
copy files from repo to-do-list

### PR DESCRIPTION
**Pull request by eduardosancho**
# What was done in this PR: 🇨🇷 

- [x] Copy files from the to-do list repo.
- [x] Stick to DRY guidelines.
- [x] Stick to KISS guidelines.
- [x] Stick to YAGNI guidelines.

mercy

### CODE REVIEW

File: src/modules/ToDoList.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    oldRows.forEach((notSelected) => {
+      notSelected.classList.remove('yellow');
+    });
+
+    row.classList.toggle('yellow');
+    row.querySelector('div').innerHTML = `
+        <form id="update-task-form-${index}">
+        <input type="text" class="update-task-description" id="update-task-description-${index}" value="${oldDescription}" autofocus>
+        </form>
+        `;
```
Comment (Yothu) [inline src/modules/ToDoList.js:94]: #### Keep It Simple, Smart
Every time that you press a task it changes the description tag between being a `<p>` and a `<input>`, I would recommend to just sticking with it being a `<input>` all the time. That way you reduce code lines and save DOM resources.

**Review by Yothu — COMMENTED** [review]
## Peer review
Good work so far!
But there is a minor point to address before merging!

#### Don't repeat yourself (DRY)
I think you completely follow this rule, you put an appropriate name to every variable.
#### Keep it simple, smart (KISS)
I think you need to do some minor tweaks to follow this rule, I think there is no need to change every task description tag from `<p>` to `<input>` every time you click on it, just keep it simple and use the `<input>` tag directly in the tasks descriptions.
#### You Aren't Gonna Need It (YAGNI)
I think you completely follow this rule, you don't add unnecessary functions.
### Summary
In conclusion, you are doing absolutely great so far, just not over-complicate the solutions.

File: src/modules/ToDoList.js (hunk 1 above)
Comment (eduardosancho) [reply src/modules/ToDoList.js:94]: Great insight!
Thanks for pointing it out, now I noticed that a big problem I encountered was caused because I changed the `<p>` to a `<input>`, using an input all the time might solve that.
